src/lib/edje/edje_private.h

```diff
@@ -74,17 +74,17 @@
 #ifdef _WIN32
 # ifdef EFL_BUILD
 #  ifdef DLL_EXPORT
-#   define EAPI __declspec(dllexport)
```

Comment:
This probably should be deleted to include `edje_api`

Reply:
yeah it would make more sense